src/js/recommendation-ui.js

```
@@ -118,19 +118,43 @@ document.addEventListener('DOMContentLoaded', () => {
-      }).catch(err => {
-        console.error("File Read Error:", err);
-        showError("Failed to read file. Please make sure it's a valid text format.");
-      });
-    });
-  }
-
+      }
+      // PDF FILE SUPPORT
+      else if (file.type === "application/pdf") {
+        const arrayBuffer = await file.arrayBuffer();
+        const pdf = await pdfjsLib.getDocument({
```

ESLint

[error] 'pdfjsLib' is not defined.

(no-undef)